### PR TITLE
refactor: hub-executor-batching

### PR DESCRIPTION
**Pull request by Kelton8Z**

### CODE REVIEW

**Review by nan-wang — APPROVED** [review]
LGTM👍

**Review by nan-wang — CHANGES_REQUESTED** [review]
bump the version in the `manifest.yml` for each pod

File: encoders/image/FaceNetEncoder/manifest.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ author: Anton Kiselev (straw.berry.pie@ya.ru)
 url: https://jina.ai
 vendor: Jina AI Limited
 documentation: https://github.com/jina-ai/jina-hub/blob/master/encoders/image/FaceNetEncoder/README.md
-version: 0.0.14
+version: 0.0.2
```
Comment (JoanFM) [inline encoders/image/FaceNetEncoder/manifest.yml:11]: it should be 0.0.15

File: encoders/image/TirgImageEncoder/manifest.yml [hunk 2]
```diff
@@ -8,6 +8,6 @@ author: Jina AI Dev-Team (dev-team@jina.ai)
 url: https://jina.ai
 vendor: Jina AI Limited
 documentation: https://github.com/jina-ai/jina-hub
-version: 0.0.14
+version: 0.0.6
```
Comment (JoanFM) [inline encoders/image/TirgImageEncoder/manifest.yml:11]: 0.0.15

File: encoders/multimodal/TirgMultimodalEncoder/manifest.yml [hunk 3]
```diff
@@ -8,6 +8,6 @@ author: Jina AI Dev-Team (dev-team@jina.ai)
 url: https://jina.ai
 vendor: Jina AI Limited
 documentation: https://github.com/jina-ai/jina-hub
-version: 0.0.14
+version: 0.0.7
```
Comment (JoanFM) [inline encoders/multimodal/TirgMultimodalEncoder/manifest.yml:11]: 0.0.15

File: encoders/nlp/CLIPTextEncoder/manifest.yml [hunk 4]
```diff
@@ -8,7 +8,7 @@ description: |
 url: https://jina.ai
 vendor: Jina AI Limited
 documentation: https://github.com/jina-ai/jina-hub
-version: 0.0.14
+version: 0.0.3
```
Comment (JoanFM) [inline encoders/nlp/CLIPTextEncoder/manifest.yml:11]: 0.0.15

File: rankers/LightGBMRanker/manifest.yml [hunk 5]
```diff
@@ -7,7 +7,7 @@ author: Jina AI Dev-Team (dev-team@jina.ai)
 url: https://jina.ai
 vendor: Jina AI Limited
 documentation: https://github.com/jina-ai/jina-hub/blob/master/rankers/LightGBMRanker/README.md
-version: 0.0.14
+version: 0.0.3
```
Comment (JoanFM) [inline rankers/LightGBMRanker/manifest.yml:10]: 0.0.15

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: encoders/multimodal/TirgMultimodalEncoder/__init__.py [hunk 6]
```diff
@@ -64,7 +64,7 @@ def _get_features(self, data):
 
         return self.model.compose_img_text_features(img_features, text_features)
 
-    @batching_multi_input(num_data=2)
+    @batching(batch_size=2)
```
Comment (JoanFM) [inline encoders/multimodal/TirgMultimodalEncoder/__init__.py:67]: is not batch size, is slice_nargs

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: encoders/image/FaceNetEncoder/manifest.yml (hunk 1 above)
Comment (nan-wang) [reply encoders/image/FaceNetEncoder/manifest.yml:11]: It is ok to just `+1`, right? @JoanFM

**Review by nan-wang — CHANGES_REQUESTED** [review]
update the `jina` version in the corresponding Dockfiles to `1.1.5`

**Comment by JoanFM** [comment]
> update the `jina` version in the corresponding Dockfiles to `1.1.5`

This should be done by dependabot rightm

File: encoders/multimodal/TirgMultimodalEncoder/__init__.py (hunk 6 above)
Comment (Yongxuanzhang) [reply encoders/multimodal/TirgMultimodalEncoder/__init__.py:67]: This reminds me of how to config batch_size for these encoders? It seems that we don't expose this param at the moment? @JoanFM

**Review by JoanFM — APPROVED** [review]

**Review by JoanFM — APPROVED** [review]

**Pull request merged**